woocommerce/wc-admin
# Tracks: Don't track if Jetpack is not present or in dev env.

One small follow-up to #452 based on some conversations and feedback on prior PRs. This branch adds Jetpack being installed/activated as a requirement to the logic to determine if tracking is enabled ( per @warrendholmes ), it also adds a check to determine if the node process is `development` and if so, not to record tracks so we don't gather data from our development usage.

__To Test__
- Open up an analytics page and verify no `t.gif` settings are recorded
- Comment out the changes in `client/lib/tracks.js`, or start up your node env in non-development mode
- Verify that `t.gif` is being recorded on analytics pages
- Disable Jetpack
- Verify that `t.gif` is not being recorded.
